RCSwitch.cpp

```diff
@@ -97,7 +96,7 @@ void RCSwitch::setProtocol(Protocol protocol) {
   * Sets the protocol to send, from a list of predefined protocols
   */
 void RCSwitch::setProtocol(int nProtocol) {
-  if (nProtocol < 1 || nProtocol > numProto) {
+  if (nProtocol < 1 || nProtocol > (int)COUNT_OF(proto)) {
```

Comment:
COUNT_OF is not a standard macro, so I wouldn't want to use it. What's wrong with `numProto`? The only argument against it I can think of is that it wastes 2 bytes of flash -- while I think the compiler will actually optimize it away anyway, we could make sure of that by turning it into an enum.

However, considering that this patch adds two bytes *per pulse pair, so 6 bytes per protocol, I don't think that's what you had in mind here... or is it? :-)